journal/modernized/13-chapter-12.adoc

```diff
@@ -378,14 +381,15 @@ because this would hinder them from harbouring their kindred and countrymen,
 who were raparees, as they had frequently done before,
 therefore they got eight or nine bloody raparees,
 to lie in ambush between Mount-melick and Rossenallis in order to kill me,
+//Hyphenated here too??
```

Comment:
yeah, just leave it here too.  We'll figure it out and change them all at once with the linter.